tsfel/feature_extraction/features.py

```diff
@@ -337,6 +338,78 @@ def neighbourhood_peaks(signal, n=10):
     return np.sum(peaks)
 
 
+@set_domain("domain", "temporal")
+def lempel_ziv(signal, threshold=None):
+    """Computes the Lempel-Ziv's (LZ) complexity index.
+    
+    Parameters
+    ----------
+    signal : np.ndarray
+        Input signal.
+    amp_thres : float, optional
+        Amplitude Threshold for the binarisation. If None, the mean of the signal is used.
+
+    Returns
+    -------
+    lz_index : float
+        Lempel-Ziv complexity index
+    """
+    if threshold is None:
+        threshold = np.mean(signal)
+    
+    binary_signal = (signal > threshold).astype(int)
+    lz_index = LZ76(binary_signal)
+
+    return lz_index
+
+
+@set_domain("domain", "temporal")
+def mse(signal, m=3, maxscale=None, tolerance=None):
+    """Computes the Multiscale entropy (MSE) of the signal, that performs the entropy
+    analysis over multiple time scales.
+
+    Parameters
+    ----------
+    signal : np.ndarray
+        Input signal.
+    m : int
+        Embedding dimension for the sample entropy, defaults to 3.
+    maxscale : int
+        Maximum scale factor, defaults to 1/13 of the length of the input signal.
+    tolerance : float
+        Tolerance value, defaults to 0.2 times the standard deviation of the input signal.
+
+    Returns
+    -------
+    mse_area : np.ndarray
+        Normalized area under the MSE curve.
+    """
+
+    if np.var(signal) == 0 and np.all(signal == signal[0]):
+        return np.nan
+
+    n = len(signal)
+
+    if n < FRACTAL_FEATURES_MIN_SIZE:
+        raise ValueError("Input signal must have at least %s data points." % FRACTAL_FEATURES_MIN_SIZE)
+
+    if tolerance is None:
+        tolerance = 0.2 * np.std(signal)
+
+    if maxscale is None:
+        maxscale = n // (10 + 3)
+        
+    mse = np.zeros(maxscale)
+    for i in range(maxscale):
```

Comment:
Maybe a more Pythonic approach?

```suggestion
    mse = [sample_entropy(coarse_graining(signal, i + 1), m, tolerance) for i in range(maxscale)]
```